http-filter-example/README.md

```diff
@@ -0,0 +1,38 @@
+# Envoy filter example
+
+This project demonstrates the linking of additional http filters with the Envoy binary.
```

Comment:
Nit: I prefer `HTTP` rather than `http` everywhere that this is used as a standalone acronym.